lib/tasks/evm_dba.rake

```diff
@@ -132,42 +64,22 @@ namespace :evm do
       exit 1
     end
 
+    # this is used by the appliance console to create a db
     # Example usage:
-    #   RAILS_ENV=production bin/rake evm:db:region -- --region 99
+    #   RAILS_ENV=production REGION=99 bin/rake evm:db:region
+    # Alt usage:
+    #   RAILS_ENV=production REGION=99 VERBOSE=false bin/rake db:reset db:seed
 
     desc 'Set the region of the current ManageIQ EVM Database (VMDB)'
-    task :region do
-      opts = EvmDba.with_options do
-        opt :region, "Region number", :type => :integer, :required => ENV["REGION"].blank?
-      end
```

Comment:
Is this the the last caller of `with_options`?  Are all of the other deletions in `EvmDba` no longer called?

Reply:
and the last caller into `EvmDba`